python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -939,6 +939,12 @@ def aten〇narrow〇Tensor〡shape(self: List[int], dim: int, start: List[int],
 def aten〇slice_scatter〡shape(self: List[int], src: List[int], dim: int = 0, start: Optional[int] = None, end: Optional[int] = None, step: int = 1) -> List[int]:
     return self
 
+def aten〇masked_scatter〡shape(self: List[int], mask: List[int], source: List[int]) -> List[int]:
+    return self
+
+def aten〇masked_scatter_〡shape(self: List[int], mask: List[int], source: List[int]) -> List[int]:
```

Comment:
no need to add underscore variant, since torch-mlir will replace it with the regular one